ros-colcon-build/melodic/azure-pipelines.yml

```diff
@@ -16,18 +16,18 @@ jobs:
     BUILD_TOOL_PACKAGE: 'ros-colcon-tools'
     BUILD_TOOL_PACKAGE_VERSION: '0.0.1.1905240527'
     PYTHON_LOCATION: 'c:\opt\python37amd64'
+    ROS_PYTHON_VERSION: '3'
+    OS_DISTRO: 'melodic'
```

Comment:
please check if this is a typo

Reply:
Good catch! You just save the build today. :)